ci/gpu_test.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -e
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+gpuci_logger "Get env"
+env
+
+gpuci_logger "Activate conda env"
+. /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
+
+echo "CUDA version for mamba install: ${CUDA}"
+
+gpuci_logger "Check versions"
+python --version
+$CC --version
+$CXX --version
+
+gpuci_logger "Check conda environment"
+conda info
+conda config --show-sources
+conda list --show-channel-urls
+
+
+# FIX ME: These paths are to be dynamically computed based on env vars and vary per build type.
+# We will have a utility tool that consolidates the logic to compute the correct paths.
+CPP_FILE_NAME="ci/rmm/pull-request/${CHANGE_ID}/${GIT_COMMIT}/librmm_${ARCH}.tar"
+PY_FILE_NAME="ci/rmm/pull-request/${CHANGE_ID}/${GIT_COMMIT}/rmm_${ARCH}.tar"
+
+aws s3 cp "s3://rapids-downloads/ci/${CPP_FILE_NAME}" conda_cpp.tar
+aws s3 cp "s3://rapids-downloads/ci/${PY_FILE_NAME}" conda_py.tar
+ls -al
+mkdir -p cpp__artifact py__artifact
+tar -xvf conda_cpp.tar -C cpp__artifact/
+tar -xvf conda_py.tar -C py__artifact/
+
+gpuci_mamba_retry install -y \
+    -c ./cpp__artifact/.conda-bld \
+    -c ./py__artifact/.py-conda-bld \
+    rmm librmm librmm-tests
+
+TESTRESULTS_DIR=${WORKSPACE}/test-results
```

Comment:
```suggestion
TESTRESULTS_DIR=test-results
```

Let's see if removing `$WORKSPACE` from here works. This will help when users are running the scripts locally, since `WORKSPACE` is only defined by Jenkins